### ISSUE TITLE
Possible fix for stremio's stuttering due to subtitles in VLC 

### ISSUE THREAD

**Issue by cerels** [Stremio/stremio-bugs#702]
**Describe the bug**
Stremio VLC player stutters every time s subtitle line appears, seems to happen mostly on .ssa subtitles which are mostly used in anime 

**To Reproduce**
You play any media that uses .ssa subtitles and notice the stutter every single time a line appears 

**Expected behavior**
It should just play normally

**TV (please complete the following information):**
 - Device: onn google tv 4k 2023
 - OS: google tv
 - Version Android 12

**Additional context**

So I found this [forum](https://forum.videolan.org/viewtopic.php?t=144325&start=50#p527307) from VLC talking about this bug, and one poster I think found a possible fix for this

> I have Amazon Fire TV Lite with VLC 3.4.4 and have this frame drop on every subtitle change.
What helped for me is set OpenGL ES2 to "force on" in Advanced settings, and Close And Restart VLC.
Founding this take me about 6 hour fighting with reading logs, reinstalling different versions, fiddling with settings etc.
I have consistent results: when I set this to Always Off - subtitle frame drop comes back. When I switch it to Force On - subtitle frame drop disappears.
When I set it to Automatic then sometimes subtitles cause frames drop, sometimes not.

**Comment by bankov4eto** [comment]
We have discussed this issue but with no success. I hope it gets fixed sometime. https://github.com/Stremio/stremio-bugs/issues/672

**Comment by razeNFR** [comment]
Yeah same problem, i use libvlc on my shield tv because exoplayer had some problem, and when subtitle appear the video stutter
